README.md

```diff
@@ -16,11 +16,12 @@ The config object passed in must have the following sections and fields:
 ```js
 var config = {
   secret: 'The client secret, only needed for OAuth v1',
-  clientId: 'Your own client id',
+  clientId: 'Your own client id, only needed for OAuth v1',
   domain: 'The token issuer w/o https:// or the trailing /',
   realm: 'The realm of the token, used in the challenge headers, full url (w/ https:// and  trailing /)',
   jwksUrl: 'The url to retrieve the jwk from, only needed for OAuth v2',
   audience: 'The audience for the auth token',
+  disableV1: 'Should auth v1 be enabled (defaults to false), not required',

```

Comment:
We're the only existing clients and we want to disable v1! You make a good point though, in that case it should still be enableV1 defaulting to true instead of disableV1 defaulting to false